receiver/azureeventhubreceiver/azure_formatter_impl.go

```diff
@@ -0,0 +1,204 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package azureeventhubreceiver // import "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/azureeventhubreceiver"
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"strconv"
+
+	"github.com/relvacode/iso8601"
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/pdata/pcommon"
+	"go.opentelemetry.io/collector/pdata/plog"
+)
+
+const azureCategory = "azure.category"
+const azureCorrelationID = "azure.correlation.id"
+const azureDuration = "azure.duration"
+const azureIdentity = "azure.identity"
+const azureOperationName = "azure.operation.name"
+const azureOperationVersion = "azure.operation.version"
+const azureProperties = "azure.properties"
+const azureResourceID = "azure.resource.id"
+const azureResultType = "azure.result.type"
+const azureResultSignature = "azure.result.signature"
+const azureResultDescription = "azure.result.description"
+const azureTenantID = "azure.tenant.id"
+
+const cloudProvider = "cloud.provider"
+const cloudRegion = "cloud.region"
+
+const netSockPeerAddr = "net.sock.peer.addr"
```

Comment:
These should be defined in https://github.com/open-telemetry/opentelemetry-collector/tree/main/semconv/v1.9.0, could I ask you to use them instead?

Reply:
Done.